### ISSUE TITLE
Option for excluding directories

### ISSUE THREAD

**Issue by nkprince007** [dave/courtney#9]
While using the `godep` tool, a `vendor` subdirectory is created. It should be possible to exclude that from tests.

**Comment by dave** [comment]
Yes this is true. Sounds like this should be the default behaviour?
